### ISSUE TITLE
pquery: --attr=revision outputs -r0 as revision="MISSING"

### ISSUE THREAD

**Issue by mgorny** [pkgcore/pkgcore#225]
Not sure if it's incorrect but it at least looks weird.

```
$ pquery -r gentoo --attr version --attr revision
app-accessibility/SphinxTrain-0.9.1-r1 version="0.9.1" revision="1"
app-accessibility/SphinxTrain-1.0.8 version="1.0.8" revision="MISSING"
app-accessibility/accerciser-3.14.0 version="3.14.0" revision="MISSING"
app-accessibility/accerciser-3.14.0-r1 version="3.14.0" revision="1"
